index.js

```diff
@@ -181,3 +185,53 @@ AddonManager.addInstallListener({
     app.send('addon-install:download-failed', formatInstallData(install));
   }
 });
+
+function pingServer(title, addon) {
+  cookieManager.getCookiesFromHost(prefs.HOSTNAME, function(cookies) {
+    let csrf;
+    let cookieStr = '';
+    cookies.forEach(function(c) {
+      cookieStr += c.name + '=' + c.value + ' ';
+      csrf = c.value;
```

Comment:
I think you probably want to check that `c.name === 'csrftoken'` here, before assigning `csrf = c.value`